job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContextSpec.groovy

```diff
@@ -3042,21 +3043,27 @@ class PublisherContextSpec extends Specification {
             tag 'tag'
             shouldWaitForRundeckJob()
             shouldFailTheBuild false
+            shouldIncludeRundeckLogs()
         }
 
         then:
         Node rundeckNode = context.publisherNodes[0]
         rundeckNode.name() == 'org.jenkinsci.plugins.rundeck.RundeckNotifier'
-        rundeckNode.jobId[0].value() == 'jobId'
-        rundeckNode.options[0].value() == 'key1=value1\nkey2=value2\nkey4=value4\nkey3=value3'
-        rundeckNode.nodeFilters[0].value() == 'key1=value1\nkey2=value2\nkey4=value4\nkey3=value3'
-        rundeckNode.tag[0].value() == 'tag'
-        rundeckNode.shouldWaitForRundeckJob[0].value() == true
-        rundeckNode.shouldFailTheBuild[0].value() == false
+        rundeckNode.jobId[0]?.value() == 'jobId'
+        rundeckNode.options[0]?.value() == 'key1=value1\nkey2=value2\nkey4=value4\nkey3=value3'
+        rundeckNode.nodeFilters[0]?.value() == 'key1=value1\nkey2=value2\nkey4=value4\nkey3=value3'
+        rundeckNode.tag[0]?.value() == 'tag'
+        rundeckNode.shouldWaitForRundeckJob[0]?.value() == true
+        rundeckNode.shouldFailTheBuild[0]?.value() == false
+        rundeckNode.includeRundeckLogs[0]?.value() == true
+
+        and:
         1 * jobManagement.requirePlugin('rundeck')
+        1 * jobManagement.requireMinimumPluginVersion('rundeck', '3.4')
     }
 
-    def 'call rundeck with invalid jobId should fail'() {
```

Comment:
I reverted changes in all not Rundeck related tests, but as I made modifications in Rundeck code I would like to keep Rundeck related tests in a good fashion in case of regression in the future. Without `@Unroll` It is harder to see for which input data the condition failed.